ui/app/rif/pages/domainsDetailPage.js

```diff
@@ -2,40 +2,90 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { faCheckCircle, faArchive, faBolt, faChevronLeft } from '@fortawesome/free-solid-svg-icons'
+import { faCheckCircle, faArchive, faBolt, faChevronLeft, faCoins, faPlusCircle } from '@fortawesome/free-solid-svg-icons'
+import { faBitcoin, faEthereum } from '@fortawesome/free-brands-svg-icons'
 import rifActions from '../../rif/actions'
 
 class DomainsDetailScreen extends Component {
-  
+	state = {
+		resolvers: this.props.resolvers,
```

Comment:
Since props must be inmutable, isnt better to clone the value to assign it to the state?

`resolvers= Object.assign({}, this.props.resolvers);
`

Reply:
Its resolved in newer commits.